File: GiroDashboard.Application/DTOs/Muhasebe/KasiyerPerformansModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace GiroDashboard.Application.DTOs.Muhasebe
{
    public class KasiyerPerformansModel
    {
        public int? SubeKodu {  get; set; }

        public string? SubeAdi { get; set; }

        public int? Kasiyer { get; set; }

        public string? AdSoyad { get; set; }

        public string? StokAd { get; set; }

        public decimal? SatisMiktar { get; set; }

        public decimal? SatisTutar { get; set; }
    }
}

File: lib/_muhasebe/kasiyer_performans/model/kasiyer_performans_model.dart
import 'dart:convert';

import '../../../core/base/base_model.dart';
import 'package:json_annotation/json_annotation.dart';

part 'kasiyer_performans_model.g.dart';

@JsonSerializable()
class KasiyerPerformansModel extends BaseModel{
  final int? subeKodu;
  final String? subeAdi;
  final int? kasiyer;
  final String? adSoyad;
  final String? stokAd;
  final double? satisMiktar;
  final double? satisTutar;

  KasiyerPerformansModel(this.subeKodu, this.subeAdi, this.kasiyer,
      this.adSoyad, this.stokAd, this.satisMiktar, this.satisTutar);

  factory KasiyerPerformansModel.fromJson(Map<String, dynamic> json) =>
      _$KasiyerPerformansModelFromJson(json);

  Map<String, dynamic> toJson() => _$KasiyerPerformansModelToJson(this);
  
  @override
  String myJson() =>jsonEncode(toJson());
}
